cypress/integration/core-opensearch-dashboards/opensearch-dashboards/datasource-management-plugin/1_create_datasource.spec.js

```diff
@@ -23,6 +23,51 @@ const miscUtils = new MiscUtils(cy);
 const username = Cypress.env('username');
 const password = Cypress.env('password');
 
+export const CreateDataSourceNoAuth = (port) => {
```

Comment:
can we add as a cypress command, or modify the existing one here, for others to easily use?

https://github.com/opensearch-project/opensearch-dashboards-functional-test/blob/a11953a791a4bea7dff437fcabef37bcb2fe5287/cypress/utils/dashboards/datasource-management-dashboards-plugin/commands.js#L32

Reply:
Done, we may need a follow up/confirmation that this command is actually working. Dashboard not coming up due to issue in notifications dashboard. I will create an issue in that repo.